discord/abc.py

```diff
@@ -341,8 +341,7 @@ class GuildChannel:
 
         def __init__(
             self, *, state: ConnectionState, guild: Guild, data: dict[str, Any]
-        ):
-            ...
+        ): ...
```

Comment:
yeah idk about that

Reply:
I'm not a huge fan either but I don't see a problem with switching to that